modules/apps/object/object-test/src/testFunctional/macros/ObjectNotifications.macro

```diff
@@ -81,16 +81,11 @@ definition {
 	}
 
 	macro deleteQueue {
```

Comment:
Replace it with `deleteNotificationEntry`

It's clearer to understand how this macro works.